app/com/linkedin/drelephant/util/Utils.java

```diff
@@ -102,7 +102,7 @@ public static Document loadXMLDoc(String filePath) {
     Map<String, String> options = new HashMap<String, String>();
     String[] tokens = str.trim().split("\\s");
     for (String token : tokens) {
-      if (token.isEmpty()) {
+      if (token.isEmpty() || token.startsWith("-X")) {
```

Comment:
Can you elaborate on this? Maybe it's just me but I'm sightly confused about what this is far/why you need it. Is this change related to Oozie or is it something separate?